src/projects/providers/multiplex/multiplex_profile.cpp

```diff
@@ -511,12 +511,12 @@ namespace pvd
 				_last_error = ov::String::FormatString("Failed to parse url: %s", url_str.CStr());
 				return false;
 			}
-			
+
```

Comment:
I think it would be nice if there were no more edits that would add blank spaces like this.

Reply:
It is actually removing extra spaces. It is probably my editor doing it. I can keep those extra spaces, no worries.